src/sentry/api/exceptions.py

```diff
@@ -56,6 +56,17 @@ def __init__(self, organization):
         super().__init__(loginUrl=reverse("sentry-auth-organization", args=[organization.slug]))
 
 
+class MemberDisabledOverLimit(SentryAPIException):
```

Comment:
Should this live in sentry?

Reply:
ad discussed in another thread, this is OK for Sentry since it's not specific to any sort of plan (just a generic member limit).